README.md

```diff
@@ -54,13 +54,13 @@ There are two main categories of samples: **Camera** and **Applications**. The s
     - **Visualization**
       - [**CaptureFromFileCameraVis3D**][CaptureFromFileCameraVis3D-url] - Capture point clouds, with color, from the virtual Zivid camera, and visualize it. Currently supported by Zivid One.
       - [**CaptureVis3D**][CaptureVis3D-url] - Capture point clouds, with color, from the Zivid camera, and visualize it.
-      - [**CaptureWritePCLVis3D**][CaptureWritePCLVis3D-url] - Capture point clouds, with color, from the Zivid camera, save it to PCD file format, and visualize it.
+      - [**CaptureWritePCLVis3D**][CaptureWritePCLVis3D-url] - Capture point clouds, with color (and normals), from the Zivid camera, save it to PCD file format, and visualize it.
```

Comment:
I think update the description to match the `ReadPCLVis3D`, since they correspond.
Maybe `Capture point cloud with/without normals [...]` and `Read point cloud with/without normals [...]`

Reply:
done

Reply:
looks good :+1: